eng/Packages.Data.props

```diff
@@ -66,8 +66,8 @@
     <PackageReference Update="Microsoft.CodeAnalysis" Version="2.3.0" />
     <PackageReference Update="Microsoft.DotNet.ApiCompat" Version="5.0.0-beta.20467.1" />
     <PackageReference Update="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="4.5.1" />
-    <PackageReference Update="Microsoft.Identity.Client" Version="4.16.1" />
-    <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="2.12.0" />
+    <PackageReference Update="Microsoft.Identity.Client" Version="4.22.0" />
```

Comment:
This was requested, but is there due cause to update? It's quite a jump, so I assume a plethora of bug fixes, but doesn't that potentially cause other dependency problems that seem to plague MSAL?

Reply:
We typically upgrade to the latest MSAL (and MSAL extension library) prior to our GA releases of Azure.Identity.